acra-censor/handlers/blacklist_handler.go

```diff
@@ -188,17 +188,25 @@ func (handler *BlacklistHandler) Release() {
 	handler.Reset()
 }
 
-// AddQueries adds queries to the list that should be blacklisted
+// AddQueries normalizes and adds queries to the list that should be blacklisted
 func (handler *BlacklistHandler) AddQueries(queries []string) {
 	for _, query := range queries {
-		handler.queries[query] = true
+		normalizedQuery, _, err := NormalizeAndRedactSQLQuery(query)
```

Comment:
why we use redacted queries here? what if we will need block all queries `where user_id=1` or `where is_superuser=True`? it will add `where user_id=::replaced::` as I understand

Reply:
it's not redacted, it's normalized.

to make `where user_id=1` same as `WHERE user_id=1`